app.js

```diff
@@ -84,5 +85,10 @@ addNunjucksFilters(env)
 nunjucks.installJinjaCompat()
 
 app.set('view engine', 'njk')
+app.use('/', function (req, res, next) {
```

Comment:
I'm not sure of the syntax, but could you see if there's a way to use this middleware on all requests instead of just `/`?